Bottom: 4e514ca44f6af4555b54ce0a6ff9b4216972a270
Top:    69bb608078fe340fef83f047822e302575ab4fa4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:26:52 -0500

Create new play for docker-hosts tasks

The majority of the docker container tasks belong to daedalus, while the
common docker tasks are for all docker hosts.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index b7bee5ae..7cd187c5 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -18,7 +18,8 @@
   - role: bacula-script
     assemble_scripts: false
   - role: bacula-catalog
-- name: Install/Setup Docker images and containers
+
+- name: Prepare Docker host systems
   hosts: docker-hosts
   remote_user: ansibleremote
   become: true
@@ -33,6 +34,14 @@
     role: docker-common
     tags:
     - docker_common
+
+- name: Install/Setup Docker images and containers on daedalus
+  hosts: daedalus
+  remote_user: ansibleremote
+  become: true
+  become_user: root
+  become_method: sudo
+  roles:
   - name: Configure Docker networking
     role: docker-networking
     net_connection: '{{ daedalus_net_connection }}'
@@ -90,6 +99,10 @@
 
 - name: Setup the Build System (Test Station) with repo data
   hosts: buildsystem
+  remote_user: ansibleremote
+  become: true
+  become_user: root
+  become_method: sudo
   roles:
   - role: buildsystem
     tags:
